python/chronos/test/bigdl/chronos/forecaster/tf/test_lstm_keras_forecaster.py

```diff
@@ -95,6 +95,31 @@ def test_lstm_forecaster_save_load(self):
         load_model_yhat = self.forecaster.predict(test_data[0])
         assert yhat.shape == (400, 1, 2)
         np.testing.assert_almost_equal(yhat, load_model_yhat, decimal=5)
+    
+    def test_lstm_customized_loss_metric(self):
+        train_data, test_data = create_data(tf_data=True)
+        loss = tf.losses.mean_squared_error
```

Comment:
Well I am not quite sure if we need to make this an tf2 loss? `tf.keras.losses.MeanSquaredError()`
reference: https://www.tensorflow.org/api_docs/python/tf/compat/v1/losses/mean_squared_error